core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -357,6 +357,12 @@ public Transaction newTransaction() {
     throw new UnsupportedOperationException(errorMsg("newTransaction"));
   }
 
+  @Override
+  public IncrementalAppendScan newIncrementalAppendScan() {
+    TableOperations ops = new StaticTableOperations(metadataFileLocation, io, locationProvider);
+    return new BaseIncrementalAppendScan(ops, lazyTable());
```

Comment:
should this be `lazyTable().newIncrementalAppendScan`?

nit: please move this method next to `newScan()` above following the same order as `Table` interface.

Reply:
Done.